c_src/ex_dtls/native.spec.exs

```diff
@@ -4,29 +4,28 @@ interface NIF
 
 state_type "State"
 
-spec init(client_mode :: bool, dtls_srtp :: bool) :: {:ok :: label, state}
+spec init(client_mode :: bool, dtls_srtp :: bool) :: state
 
-spec init_from_key_cert(client_mode :: bool, dtls_srtp :: bool, pkey :: payload, cert :: payload) :: {:ok :: label, state}
+spec init_from_key_cert(client_mode :: bool, dtls_srtp :: bool, pkey :: payload, cert :: payload) ::
+       state
 
-spec generate_cert(state) :: {{:ok :: label, cert :: payload}, state}
+spec generate_cert(state) :: {cert :: payload, state}
 
-spec get_pkey(state) :: {{:ok :: label, pkey :: payload}, state}
+spec get_pkey(state) :: {pkey :: payload, state}
 
-spec get_cert(state) :: {{:ok :: label, cert :: payload}, state}
+spec get_cert(state) :: {cert :: payload, state}
 
-spec get_cert_fingerprint(state) :: {{:ok :: label, fingerprint :: payload}, state}
+spec get_cert_fingerprint(state) :: {fingerprint :: payload, state}
 
-spec do_handshake(state) :: {{:ok :: label, packets :: payload}, state}
+spec do_handshake(state) :: {packets :: payload, timeout :: int, state}
 
-spec handle_timeout(state) :: {:ok :: label, state}
-                           | {{:retransmit :: label, packets :: payload}, state}
+spec handle_timeout(state) ::
+       {:ok :: label, state} | {:retransmit :: label, packets :: payload, timeout :: int, state}
 
-spec process(state, packets :: payload) :: {:ok :: label, state, packets :: payload}
-                                           | {(:hsk_want_read :: label), state}
-                                           | {{:hsk_packets :: label, packets :: payload}, state}
-                                           | {{:hsk_finished :: label,
-                                              client_keying_material :: payload,
-                                              server_keying_material :: payload,
-                                              protection_profile :: int,
-                                              packets :: payload}, state}
-                                           | {{:connection_closed :: label, :peer_closed_for_writing :: label}, state}
+spec process(state, packets :: payload) ::
```

Comment:
I feel like `receive` would fit here better but it overlaps with Elixir. There also are `recv`, `recv_data`, `feed`, `read`

Reply:
`handle_data`? We already have `handle_timeout`, but no strong opinions from me.